dumper/lib-ext-hive-metastore/src/main/java/com/google/edwmigration/dumper/ext/hive/metastore/HiveMetastoreThriftClient.java

```diff
@@ -153,9 +162,10 @@ public HiveMetastoreThriftClient build() throws TTransportException {
         if (unavailableClientBehavior == UnavailableClientVersionBehavior.FALLBACK) {
           LOG.warn(
               messagePrefix
-                  + " The caller requested fallback behavior, so a client compiled against a superset Thrift specification will be used instead. "
-                  + "If you encounter an error when using the fallback client, please contact CompilerWorks support and provide "
-                  + "the originally requested version number.");
+                  + " The caller requested fallback behavior, so a client compiled against a"
+                  + " superset Thrift specification will be used instead. If you encounter an error"
+                  + " when using the fallback client, please contact CompilerWorks support and"
```

Comment:
I think we should link to Google support here, not CompilerWorks.

Reply:
Added to the `TODO` list